src/test/java/com/stripe/model/ExternalAccountTest.java

```diff
@@ -59,4 +59,26 @@ public void testUnknownExternalAccountDeletion() throws StripeException, IOExcep
 			null, APIResource.RequestType.NORMAL, RequestOptions.getDefault()
 		);
 	}
+
+	@Test
+  public void testVerify() throws StripeException, IOException {
```

Comment:
And lastly, mind just converting the leading whitespace in this test to tabs as well? Thanks!